File: codewars/Js/StringEndsWith.js
function solution(str, ending) {
  if (ending.length > str.length) {
    return false;
  }
  let subs = str.substring(str.length - ending.length);
  return subs === ending;
}


function Ex2(str, ending) {
  return str.endsWith(ending);
}

const Ex3 = (s,t) => {
  return s.endsWith(t);
}

File: codewars/Java/StringEndsWith.java
public class Kata {
  public static boolean solution(String str, String ending) {
    if (ending.length() > str.length()) {
      return false;
    }
    String subs = str.substring(str.length() - ending.length());
    return subs.equals(ending);
  }
}

// Ex2
class Ex {

  boolean Solution(String str, String ending) {
    return str.endsWith(ending);
  }
}
